### PR TITLE
corrected tests to account for changes in the sort function

### PR DESCRIPTION
**Pull request by polyccon**
added tests to delete function

changed language to be homogenuous

### CODE REVIEW

File: skeleton/logic.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ var todoFunctions = {
   },
   sortTodos: function(todos, condition ){
     var array = todos.filter(function(x) {
-      return x.done == condition;
+      return x.done === condition;
```
Comment (rebecacalvoquintero) [inline skeleton/logic.js:65]: 👍

File: skeleton/test.js [hunk 2]
```diff
@@ -24,21 +52,33 @@ test('new array returned should not contain todo with id of idToDelete', functio
     description: "go for run",
     done: false
   }];
-  t.deepEqual(actual, expected, 'should delete object with id idToDelete');
+  t.deepEqual(actual, expected, "Should return a new array with all the items that don't match the idToDelete input argument");
```
Comment (rebecacalvoquintero) [inline skeleton/test.js:55]: ❤️

**Review by rebecacalvoquintero — APPROVED** [review]
well done! ;)

**Comment by rebecacalvoquintero** [comment]
resolve conflicts and we can merge ;)

**Comment by mineshmshah** [comment]
Fixed and updated file structure

**Pull request merged**